.chloggen/fix_red-metrics.yaml

```diff
@@ -0,0 +1,41 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Use new default metrics namespace/prefix for span RED metrics in Jaeger query.
+
+
+# One or more tracking issues related to the change
+issues: [1072]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  Use the new RED metrics default namespace `traces.span.metrics` for retrieval from Prometheus.
+  Since OpenTelemetry Collector version `0.109.0` the default namespace is active.
```

Comment:
Since OpenTelemetry Collector version `0.109.0` the default namespace is set to `traces.span.metrics`.